docs/overview.md

```diff
@@ -1,54 +1,120 @@
 Overview
 ========
 
-Maestro is a Kubernetes Game Room Scheduler.
+Maestro is a Kubernetes-native game server scheduler.
 
 ## Goal
 
-Have an unified system that automatically scales game rooms regardless of the protocol (TCP, UDP). This system is related to a matchmaker but does not handle the specificities of a match such as how many players fit in a room. It only deals with high level room occupation, i.e. is the room occupied or available. The rooms communicate directly with the matchmaker in order to register and unregister themselves from the matchmaking.
+Maestro goal is to provide an unified system that automatically scales game
+rooms, regardless of the transport layer protocol (TCP, UDP). It's also
+responsible for dealing with 
```

Comment:
I think there should be something after this?

Reply:
Fixed.